### ISSUE TITLE
Animation stutter on certain blocks - Skips and graphical issues

### ISSUE THREAD

**Issue by vegvel31** [anegostudios/VintageStory-Issues#3760]
### Game Version

1.19.5 (Stable)

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Been experiencing a strange bug with certain animations and animated blocks, namely the helve hammer, resonator, and pouring metal from a crucible. Have recorded the issues, so that you can more clearly see what I'm talking about. 

Pouring metal, see how the animation almost jumps when the pouring is almost finished (_Within the mold_): 
https://github.com/anegostudios/VintageStory-Issues/assets/68665759/a7df6fa2-3bb0-427d-b8e2-046eba6020da

Helve hammer has this weird blackout-stutter thingy that appears after a certain amount of rotations. Occurs at the end of the clip: 
https://github.com/anegostudios/VintageStory-Issues/assets/68665759/e06b5ccc-7451-4c16-8246-4dd7c7ae5db5

Animation of the resonator seems to skip a little bit, when the crank is almost done a full 360. Similar to the metal-pouring bug: 
https://github.com/anegostudios/VintageStory-Issues/assets/68665759/01300ea5-b88c-47ed-b18f-8918e3557826

Similar to the SSAO issue I reported earlier, I've tried adjusting several setting related to max number of frames and Vsync, both in Nvidea and VS, but haven't experienced any noteworthy change. I'm using a 144hz screen for both my stationary PC and my laptop, and have tried limiting FPS to both 60, 75 and 120, with no effect. I have also tried reverting to earlier VS version, with no change in the behavior. I've also tried altering other setting in Nvidea, but haven't found anything so far that changes the bugs. 

Unsure whether this has been reported earlier, so feel free to close the thread if deemed necessary. 

### How to reproduce

Just happens when I play normally. I haven't altered any spesific setting for this bug to occur, at least to my knowledge.

### Logs

```Log```

**Comment by Craluminum2413** [comment]
Is it 1.19.5 stable?

**Comment by vegvel31** [comment]
Yes. I'll also add that the stable version has been reinstalled during the process of testing bugs over different game versions. So this bug, alongside the SSAO one, just seems to follow me whatever game version I'm using.

**Comment by pingouyn** [comment]
Hi, I'll try to add something useful.
Witnessed the mentionned issue for metal pouring ever since I started playing back in October 23, regardless of version.  Haven't used a resonator yet, and didn't witnessed the blackout-stutter with the helve hammer - which doesn't mean it didn't occur.

The interesting thing is my monitor is 75 Hz, set at 60 Hz in kde plasma (linux).  When I set the VSync option in VS, VS sets the max fps to 75 Hz instead of 60 Hz.
VS doesn't retrieve the OS refresh rate current value, it picks it up from the monitor's manufacturing value (or video driver?).  Could it be a clue?

**Comment by Craluminum2413** [comment]
For crucible bug see #3774

**Comment by Craluminum2413** [comment]
Resonator animation will be fixed in 1.20.0-pre.7
